src/plugins/kernel_install/arch.sh

```diff
@@ -34,11 +34,21 @@ function generate_arch_temporary_root_file_system()
   local template_path
   local mkinitcpio_destination_path
   local LOCAL_KW_ETC="$KW_ETC_DIR/template_mkinitcpio.preset"
+  # mkinitcpio still the default
+  local root_file_system_tool='mkinitcpio'
+
+  if [[ ! -x "$(command -v 'mkinitcpio')" ]]; then
```

Comment:
I don't think it is a good way to define the tool to generate initramfs because I've installed mkinitcpio in my system (for example) and running mkinitcpio didn't work. maybe selecting by the distro name is more effective...